### PR TITLE
Added tests for Tag component

### PR DESCRIPTION
**Pull request by krolwofficial**

### CODE REVIEW

File: src/alto-ui/Tag/__tests__/Tag.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  });
+
+  it('should render button if passed onClick prop', () => {
+    const wrapper = getWrapper(defaultProps);
+
+    expect(wrapper.find('button')).toHaveLength(1);
+  });
+
+  it('should set correct class names', () => {
+    const wrapper = getWrapper(defaultProps);
```
Comment (barteksiudak) [inline src/alto-ui/Tag/__tests__/Tag.test.js]: use just `const wrapper = getWrapper(defaultProps).find('button');` and remove line 34

Comment (krolwofficial) [reply src/alto-ui/Tag/__tests__/Tag.test.js]: Did this and I named there the variable button:
`const button = getWrapper(defaultProps).find('button');`

File: src/alto-ui/Tag/__tests__/Tag.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        const wrapper = getWrapper(props);
+
+        expect(wrapper.find('button').exists()).toBe(true);
+        expect(wrapper.find('.Tag').prop('propMock')).toBe(true);
+        expect(wrapper.find('.className').exists()).toBe(true);
+        expect(wrapper.find('.Tag--active').exists()).toBe(true);
+        expect(wrapper.find('.Tag--button').exists()).toBe(true);
+        expect(wrapper.find('.Tag--disabled').exists()).toBe(true);
+        expect(wrapper.find('.Tag--rounded').exists()).toBe(true);
+    });
```
[the remark this answers is not in the document]
Comment (krolwofficial) [reply src/alto-ui/Tag/__tests__/Tag.test.js]: Refactored,

In second test I created `const button = getWrapper(defaultProps).find('button');`

File: src/alto-ui/Tag/__tests__/Tag.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+describe('Tag', () => {
+    
+    const getWrapper = props => shallow(<Tag {...props} />);
+    
+    it('should render without error',() => {
+        const wrapper = getWrapper({});
+
+        expect(wrapper.find('.Tag').exists()).toBe(true);
+        expect(wrapper.find('div').exists()).toBe(true);
```
[the remark this answers is not in the document]
Comment (krolwofficial) [reply src/alto-ui/Tag/__tests__/Tag.test.js]: Refactored

**Pull request merged**